dolly-backend-app/src/main/java/no/nav/dolly/service/TpsfPersonCache.java

```diff
@@ -39,6 +40,15 @@ public TpsPerson fetchIfEmpty(TpsPerson tpsPerson) {
             tpsPerson.getPersondetaljer().addAll(tpsfService.hentTestpersoner(new ArrayList<>(tpsfIdenter)));
         }
 
+        List<String> historikkIdenter = tpsPerson.getPerson(tpsPerson.getHovedperson()).getIdentHistorikk().stream()
```

Comment:
Eller skal det gjøres for alle personer i personDetaljer? Det vil vel kun gjelde for hovedperson?

Reply:
Ja vi nøyer oss med hovedperson i denne omgang. 👍